tests/foreman/cli/test_errata.py

```diff
@@ -1547,22 +1550,20 @@ def test_errata_list_by_contentview_filter(module_entitlement_manifest_org, modu
 
     :BZ: 1785146
     """
-    product = module_target_sat.api.Product(organization=module_entitlement_manifest_org).create()
+    product = module_target_sat.api.Product(organization=module_sca_manifest_org).create()
     repo = module_target_sat.cli_factory.make_repository(
         {'content-type': 'yum', 'product-id': product.id, 'url': REPO_WITH_ERRATA['url']}
     )
     module_target_sat.cli.Repository.synchronize({'id': repo['id']})
-    lce = module_target_sat.api.LifecycleEnvironment(
-        organization=module_entitlement_manifest_org
-    ).create()
+    lce = module_target_sat.api.LifecycleEnvironment(organization=module_sca_manifest_org).create()
     cv = module_target_sat.api.ContentView(
-        organization=module_entitlement_manifest_org, repository=[repo['id']]
+        organization=module_sca_manifest_org, repository=[repo['id']]
```

Comment:
```suggestion
        organization=module_sca_manifest_org, repository=[repo.id]
```

Reply:
From `cli_factory,make_repository`, do we expect a repository instance? 
Or just list of key-value pairs for attributes?
Ref - [pytest_fixtures/acrtivationkey.py@57](https://github.com/SatelliteQE/robottelo/blob/7bdd3207149d9c7a120adae177310d8f8a086aa8/pytest_fixtures/component/activationkey.py#L57)

Reply:
It'll return a Box object which can be accessed with a dot as well as a dict